app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,99 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.dto.Task;
+import com.codesoom.assignment.generators.IdGenerator;
+import com.codesoom.assignment.repositories.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * 할 일에 대한 HTTP request를 처리하는 컨트롤러다.
+ */
+@RestController
+@RequestMapping(path = "/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private final TaskRepository taskRepository;
+    private final IdGenerator taskIdGenerator;
+
+    @Autowired
+    public TaskController(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+        this.taskIdGenerator = new IdGenerator();
+    }
+
+    /**
+     * 모든 할 일의 리스트를 반환한다.
+     *
+     * @return 할 일 목록 전체를 반환한다.
+     */
+    @GetMapping
+    public List<Task> getAllTasks() {
+        return this.taskRepository.findAll();
+    }
+
+    /**
+     * 생성된 할 일의 인스턴스를 반환한다.
+     *
+     * @param task RequestBody의 content로 구성된 할 일 인스턴스다.
+     * @return 새로 생성된 할 일 인스턴스를 반환한다.
+     */
```

Comment:
JavaDoc은 좀 더 강하게 추상화한 내용을 적어봅시다. 이렇게 해보면 어떨까요?
```suggestion
    /**
     * 새로운 할 일을 생성해 리턴합니다.
     *
     * @param task 생성할 할 일 정보
     * @return 새로 생성된 할 일
     */
```

Reply:
코드를 설명하기 보다는 추상적으로 표현해 구현한 코드가 어떤 동작을 의미하는 지를 설명해야 하는 군요.